modules/core/src/lib/attribute-transition-manager.js

```diff
@@ -172,7 +179,10 @@ export default class AttributeTransitionManager {
       }
 
       if (hasChanged) {
-        this._triggerTransition(transition, settings);
+        this._triggerTransition(transition, settings, {
```

Comment:
Why do the last two parameters need to be an object, as opposed to two arguments? Is it still supposed to be a generic context mechanism? Make all arguments named parameters, or all of them non-named?